src/components/join-us/join-us.component.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { Link } from 'gatsby'
+
+import useJoinUsStyles from './join-us.styles'
+import Title from '../shared/title/title.component'
+
+import positions from './open-positions.json'
+
+export default function JoinUs () {
+  const classes = useJoinUsStyles()
+
+  return (
+    <>
+      <div className={classes.sectionWrapper}>
+        <section className={`${classes.section} ${classes.section1}`}>
+          <div className={`${classes.row} ${classes.rowReverse}`}>
+            <div className={classes.col2}>
+              <div className={classes.section1img} />
+            </div>
+            <div className={classes.col2}>
+              <div className={classes.section1Title}>
+                <Title>Join Hermez team</Title>
+              </div>
+              <p className={classes.paragraphMargin}>
+                It’s very exciting times for us here at Hermez Project.
+                Following our successful launch and expansion plans, we are
+                growing massively.
+              </p>
+              <p className={classes.paragraphMargin}>
+                You can see our list of vacancies below, but we’re always
+                looking for top talent, blockchain enthusiasts and if you think
+                you have what it takes to help us achieve our mission, please do
```

Comment:
The phrase have what it takes seems quite aggressive and might put off non-white male folks

Reply:
I've approval for this, What do you think?
_"You can see our list of vacancies below. In addition, we're always looking for top talent, blockchain enthusiasts and if this sounds interesting to you, please do get in touch – we always try to work out how we can fit great people into our team."_